docs/source/guide/pec-5-theory.myst

```diff
@@ -0,0 +1,156 @@
+
+Replacing each gate $\mathcal G_i$ with its noisy representation, we can express the ideal expectation
+value as a linear combination of noisy expectation values:
+
+$$
+\langle A \rangle_{\rm ideal}= {\rm tr}[A \mathcal U (\rho_0)] = 
+\sum_{\vec{\alpha}} \eta_{\vec{\alpha}} \langle A_{\vec{\alpha}}\rangle_{\rm noisy}
+$$ 
+
+where 
+
+$$ 
+\eta_{\vec{\alpha}} := \eta_{t, \alpha_t} \dots \eta_{2, \alpha_2} \eta_{1,\alpha_1}, 
+\quad  \langle A_{\vec{\alpha}}\rangle_{\rm noisy} :=  {\rm tr}[A \Phi_{\vec{\alpha}}(\rho_0)],
+\quad \Phi_{\vec{\alpha}} := \mathcal O_{t, \alpha_t} \circ \dots \circ \mathcal O_{2, \alpha_2} \circ \mathcal O_{1, \alpha_1}.
+$$
+
+
+The coefficients $\{ \eta_{\vec{\alpha}} \}$ form a quasi-probability distribution
+for the global circuit over the noisy circuits. Indeed it is easy to check that, at the level of super-operators,
+we have:
+
+$$ \mathcal U =  \sum_{\vec{\alpha}} \eta_{\vec{\alpha}} \Phi_{\vec{\alpha}}. $$
+
+The one-norm $\gamma$ of the global quasi-probability distribution is the product of those of the gates:
+
+$$
+\sum_{\vec \alpha} \eta_{\vec{\alpha}}=1,  \qquad  \gamma = \sum_{\vec{\alpha}} |\eta_{\vec \alpha}| = \Pi_{i=1}^{t} \gamma_i.
+$$
+
+All the noisy expectation values $\langle A_{\vec{\alpha}}\rangle_{\rm noisy}$ can be directly measured with 
+a noisy backend since they only require circuits composed of implementable noisy operations.
+In principle, by combining all the noisy expectation values, one could compute the ideal result $\langle A \rangle_{\rm ideal}$.
+Unfortunately this approach requires the execution of an exponentially large number of circuits and 
+which is typically unfeasible. 
+
+An important fact at the basis of PEC is that, for weak noise, only a small number of noisy expectation values actually
+contribute to the linear combination because many of the coefficients $\eta_{\vec \alpha}$ are negligible.
+For this reason, it is more efficient to estimate $\langle A \rangle_{\rm ideal}$ using an
+ ([importance-sampling](https://en.wikipedia.org/wiki/Importance_sampling)) Monte Carlo approach as described in the next section.
+
+## Monte Carlo estimation
+
+To apply a Monte Carlo estimation, we need to replace quasi-probabilities with positive probabilities.
+This can be achieved as follows:
+
+$$ \mathcal{G_i} = \sum_{\alpha} \eta_{i, \alpha} \mathcal{O}_{i, \alpha} 
+= \gamma_i \sum_{\alpha} p_i(\alpha) \, {\rm sgn}(\eta_{i, \alpha})\, \mathcal{O}_{i, \alpha},$$
+
+where $p_{i}(\alpha)=|\eta_{i, \alpha}|/\gamma_i$ is a valid probability distribution with respect to $\alpha$.
+
+If for each gate $\mathcal G_i$ of the circuit we sample a value of $\alpha$ from $p_{i}(\alpha)$ and we apply the corresponding noisy operation
+$\mathcal O_{i, \alpha}$, we are effectively sampling a noisy circuit $\Phi_{\vec{\alpha}}$ from the
+global probability distribution $p(\vec{\alpha})= |\eta_{\vec{\alpha}}| / \gamma$.
```

Comment:
could define \vec{alpha} more explicitly, maybe. In the previous subsection.

Reply:
Applied here: 40ae292